### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by vreuter**
- [x] Needs [`yacman 0.4`](https://github.com/databio/yacman/pull/7)

### CODE REVIEW

File: refgenconf/refgenconf.py [hunk 1]
```diff
@@ -347,6 +349,29 @@ def _invert_genomes(self):
...
+            Size of each block (in tqdm units) [default: 1].
+        tsize  : int, optional
+            Total size (in tqdm units). If [default: None] remains unchanged.
+        """
+        if tsize is not None:
+            self.total = tsize
+        self.update(b * bsize - self.n)
+
+
+def _download_url_progress(url, output_path, name):
```
Comment (vreuter) [inline refgenconf/refgenconf.py]: This is cool! :sunglasses:

File: refgenconf/refgenconf.py [hunk 2]
```diff
@@ -347,6 +351,23 @@ def _invert_genomes(self):
         return genomes
 
 
+class DownloadPB(tqdm):
```
Comment (nsheff) [inline refgenconf/refgenconf.py]: what's PB?

let's just call it ProgressBar or something.

**Comment by nsheff** [comment]
we haven't released any version yet so just keep these going into 0.1.0 until we have a stable first release...

**Comment by vreuter** [comment]
> we haven't released any version yet so just keep these going into 0.1.0 until we have a stable first release...

How will the tag be handled?

**Comment by nsheff** [comment]
i"ll take care of it.

**Pull request merged**